app/experimenter/nimbus-ui/src/components/PageSummary/index.test.tsx

```diff
@@ -85,16 +85,10 @@ describe("PageSummary", () => {
     ).not.toBeInTheDocument();
   });
 
-  it("renders the cancel review button if the experiment is in review state", async () => {
```

Comment:
This was a duplicate test, it's already handled in the Summary and ChangeApprovalOperations tests

Reply:
Leaving the camp site cleaner than you found it 🙏 🔥